src/App.jsx

```diff
@@ -14,70 +15,62 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {
+          selectedGood === ''
+            ? (
+              'No goods selected'
+            )
+            : (
+              <>
+                { `${selectedGood} is selected` }
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+                <button
+                  onClick={() => {
+                    setGood('');
+                  }}
+                  data-cy="ClearButton"
+                  type="button"
+                  className="delete ml-3"
+                />
+              </>
+            )
+        }
+      </h1>
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
+      <table className="table">
+        <tbody>
+          {goods.map((good, i) => (
+            <tr
+              data-cy="Good"
+              className={`${selectedGood === good && 'has-background-success-light'}`}
+              key={good}
             >
-              +
-            </button>
-          </td>
+              <td>
+                <button
+                  onClick={() => {
+                    setGood(selectedGood === good ? '' : good);
+                  }}
```

Comment:
selectedGood === good uses five times. better to make write it in variable and use in all places
```
const isSelected = selectedGood === good;
```
```suggestion
                  onClick={makeSetGood(isSelected ? '' : good)}
```